sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorCustomizer.java

```diff
@@ -19,9 +18,9 @@
 /**
  * Class to enable Azure Monitor for OpenTelemetry autoconfiguration.
  */
-public final class AzureMonitor {
+public final class AzureMonitorCustomizer {
```

Comment:
I sort of like the `AzureMonitor` as the entry point, let's chat about this further